core/digital-twin-lib/src/DAppTwin.ts

```diff
@@ -177,14 +177,10 @@ class DAppTwin extends DigitalTwin {
    * Load basic twin information and setup dispatcher watchers for loading states.
    */
   public async initialize(): Promise<void> {
+    await this.loadBaseInfo();
+    this.ownerName = await profileUtils.getUserAlias(this.runtime, this.ownerAddress);
+    this.favorite = await this.isFavorite();
     await Promise.all([
```

Comment:
Should we move these two also in the `Promise.all` and only run `loadBaseInfo` first? 

```suggestion
    await Promise.all([
      (async (): Promise<void> => {
        this.favorite = await this.isFavorite();
      })(),
      (async (): Promise<void> => {
        this.ownerName = await profileUtils.getUserAlias(this.runtime, this.ownerAddress);
      })(),
```

Reply:
I didn't like how hard those anonymous async functions are to read. 
I'm not sure if the gained performance is worth the ugly code :D